<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bmidialog</class>
 <widget class="QDialog" name="bmidialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Body mass index calculator</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelBMI">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>22</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>BMI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditHeight">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelHt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>cm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditWeight">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelWt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>bmidialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>bmidialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
